rhel/SPECS/prometheus.spec

```diff
@@ -348,21 +91,6 @@ fi
 /usr/lib/systemd/system/%{repo}.service
 %dir %attr(-, nobody, nobody) %{_sharedstatedir}/%{repo}
 
-%if 0%{?with_devel}
-%files devel -f devel.file-list
-%copying LICENSE
-%doc CHANGELOG.md CONTRIBUTING.md README.md
-#doc Godeps/Godeps.json
-%dir %{gopath}/src/%{provider}.%{provider_tld}/%{project}
-%dir %{gopath}/src/%{import_path}
-%endif
-
-%if 0%{?with_unit_test}
-%files unit-test -f unit-test.file-list
-%copying LICENSE.md
-%doc README.md
-%endif
-
 %changelog
```

Comment:
Probably worth updating?

Reply:
If you talk about changelog, it is not critical when we commit all changes. The changelog should be used if the team does not use distributed version control system at all (or commit all changes in spec files only once before the release). In our case it is ok to skip adding all changes to changelog.

Reply:
Then should we remove it?